webpack/helpers.js

```diff
@@ -0,0 +1,47 @@
+var path = require('path');
+var root = path.resolve(__dirname, '..');
+var webpack = require('webpack');
+
+exports.installVendorDLL = function(config, dllName) {
+  // DLL shizzle. Read more about this in /webpack/dlls/README.md
+  if (process.env.WEBPACK_DLLS === '1') {
+    var manifest = loadDLLManifest(path.join(root, 'webpack/dlls/manifests/' + dllName + '.json'));
+
+    if (manifest) {
+      console.warn('Webpack: will be using the "%s" DLL.', dllName);
+
+      config.plugins.push(new webpack.DllReferencePlugin({
+        context: root,
+        manifest: manifest
```

Comment:
I think it's what webpack expects; an object (the manifest) and not a path to where a manifest could be found. I could be wrong though.